guava/src/main/java/com/fasterxml/jackson/datatype/guava/deser/RangeDeserializer.java

```diff
@@ -193,6 +216,42 @@ public Range<?> deserialize(JsonParser p, DeserializationContext context)
         }
     }
 
+    private Range<?> deserializeRangeFromString(DeserializationContext context, JsonParser p)
+            throws IOException
+    {
+        String rangeInterval = p.getText();
+
+        if (rangeInterval == null || rangeInterval.isEmpty())
```

Comment:
Never returns `null` when current token is `JsonToken.VALUE_STRING`.

Reply:
Understood. If the token is a String, it can’t be null, but it could be empty, correct? If that’s the case, what would be the appropriate way to handle it?

Reply:
Correct, may be empty, just not `null`. So you can leave out the first part of check that's all.